
project build/
diff --git a/build/target/product/core.mk b/build/target/product/core.mk
index 519dbb8..c4af48b 100644
--- a/build/target/product/core.mk
+++ b/build/target/product/core.mk
@@ -26,6 +26,7 @@ PRODUCT_PACKAGES += \
     Calculator \
     Calendar \
     CalendarProvider \
+    Camera2 \
     CaptivePortalLogin \
     CertInstaller \
     Contacts \
diff --git a/build/target/product/sdk_base.mk b/build/target/product/sdk_base.mk
index 85a1607..f5b0fae 100644
--- a/build/target/product/sdk_base.mk
+++ b/build/target/product/sdk_base.mk
@@ -27,7 +27,7 @@ PRODUCT_PACKAGES := \
 	Fallback \
 	Gallery \
 	GestureBuilder \
-	LegacyCamera \
+	Camera2 \
 	librs_jni \
 	libwnndict \
 	libWnnEngDic \
