
project vendor/eos/
diff --git a/vendor/eos/config/common.mk b/vendor/eos/config/common.mk
index 71d5f2b..4f12ddc 100644
--- a/vendor/eos/config/common.mk
+++ b/vendor/eos/config/common.mk
@@ -254,7 +254,7 @@ ifdef EOS_BUILDTYPE
     endif # eos build type release or nightly
 else
     # If EOS_BUILDTYPE is not defined, set to UNOFFICIAL
-    EOS_BUILDTYPE := UNOFFICIAL
+    EOS_BUILDTYPE := ShevT
     EOS_EXTRAVERSION :=
 endif
 
