
project vendor/eos/
diff --git a/vendor/eos/config/prebuilt_apps_common.mk b/vendor/eos/config/prebuilt_apps_common.mk
index 6efb23a..ae317d1 100644
--- a/vendor/eos/config/prebuilt_apps_common.mk
+++ b/vendor/eos/config/prebuilt_apps_common.mk
@@ -8,4 +8,5 @@ PRODUCT_COPY_FILES += \
 
 # Terminal Emulator prebuilt library
 PRODUCT_COPY_FILES +=  \
-    vendor/eos/prebuilt/common/lib/libjackpal-androidterm4.so:system/lib/libjackpal-androidterm4.so \
+    vendor/eos/prebuilt/common/lib/libjackpal-androidterm5.so:system/lib/libjackpal-androidterm5.so \
+    vendor/eos/prebuilt/common/lib/libjackpal-termexec2.so:system/lib/libjackpal-termexec2.so
